tests/dummy/app/templates/identity/community-header.hbs

```diff
@@ -37,31 +37,106 @@
 	<section>
 		<h2 id="anon">Logged out with wordmark</h2>
 		{{#component-demo name='community-header-logged-out' codeOnly=true componentName='wds-community-header' standalone=true variantUrl='/identity/community-header/standalone/anon'}}
-			{{wds-community-header model=model.anon}}
+			{{#wds-community-header model=model.anon}}
+				<div class="wds-community-header__counter">
```

Comment:
I wonder if it would make sense to use multiple yields for this and `wds-community-header__wiki-buttons` :thinking: 
See https://discuss.emberjs.com/t/using-yield-for-multiple-nested-components/9126/4 for an example.

It would be something like this:
```
{{#wds-community-header model=model.anon as |community-header|}}
  {{#community-header.counter}}<span>...</span>{{/community-header.counter}}
  {{#community-header.buttons}}{{#each model.wiki-buttons as |button|}}...{{/each}}{{/community-header.buttons}}
{{/wds-community-header}}
```

This way we would force apps to use only what's available in DS. But maybe it's an overkill :man_shrugging: